mp4.go

```diff
@@ -268,9 +274,10 @@ func readCustomAtom(r io.ReadSeeker, size uint32) (_ string, data []string, _ er
 		return "", nil, err
 	}
 
-	if subNames["mean"] != "com.apple.iTunes" || subNames["name"] == "" || len(data) == 0 {
+	if !means[subNames["mean"]] || subNames["name"] == "" || len(data) == 0 {
```

Comment:
in testing, removing the `subNames["mean"] != "com.apple.iTunes"` entirely also works, but I can imagine that would cause side effects on other files.

Reply:
Agreed.  Let's keep it as specific as possible (I'm not familiar with the format/variants enough to make a better call!)